libs/web3/src/lib/use-vega-transaction-toasts.tsx

```diff
@@ -38,19 +38,20 @@ import {
   truncateByChars,
 } from '@vegaprotocol/utils';
 import { t } from '@vegaprotocol/i18n';
-import { useAssetsDataProvider } from '@vegaprotocol/assets';
+import { useAssetsMapProvider } from '@vegaprotocol/assets';
 import { useEthWithdrawApprovalsStore } from './use-ethereum-withdraw-approvals-store';
 import { DApp, EXPLORER_TX, useLinks } from '@vegaprotocol/environment';
 import {
   getOrderToastIntent,
   getOrderToastTitle,
   getRejectionReason,
   useOrderByIdQuery,
+  useStopOrderByIdQuery,
 } from '@vegaprotocol/orders';
-import { useMarketList } from '@vegaprotocol/markets';
+import { useMarketsMapProvider } from '@vegaprotocol/markets';
 import type { Side } from '@vegaprotocol/types';
 import { OrderStatusMapping } from '@vegaprotocol/types';
-import { Size } from '@vegaprotocol/datagrid';
+import { Size, formatTrigger } from '@vegaprotocol/datagrid';
```

Comment:
I see that size is already imported here.. but for me the web3 lib and more specifically the toast component should not be depending on components from the datagrid lib.

I'd move the formatTrigger function to utils